frontend/test/index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
```

Comment:
Perhaps we should have a more specific folder name than test. Our first feature is bill splitting so maybe bill page or otherwise?

Reply:
I agree but in this case probably doesn't matter considering the front end is yet to be set up. File will most lilely be deleted in future.

Reply:
Can one of you please flag the changes as approved so we can merge?